### ISSUE TITLE
PyTorch model has poor performance at high level

### ISSUE THREAD

**Issue by lliuz** [NVlabs/PWC-Net#38]
Hi @deqings,
I have tested the pre-trained models provided on Sintel and FlyingChairs datasets. The result of flow2 looks good. However, the results at the higher levels have worse performance. Even at the top level of flow6, the scale of prediction is not on same the order of magnitude. 
```sh
size:  [(1L, 2L, 112L, 256L), (1L, 2L, 56L, 128L), (1L, 2L, 28L, 64L), (1L, 2L, 14L, 32L), (1L, 2L, 7L, 16L)]
max: [1.894425392150879, 1.697497010231018, 1.6027228832244873, 1.060249924659729, 0.23016595840454102]
min: [-0.40163561701774597, -0.5419626832008362, -1.1349056959152222, -0.5822294354438782, -0.2260863333940506]
```
I don't know if this is normal. I have tested on pytorch0.4.1/python3.5 and pytorch0.2.3/python2.7. Hope anyone can help me.

**Comment by luvegood** [comment]
The flow is divided by 20.0 in the training, you may check the section of "stuffs that may confuse you" in the readme: https://github.com/NVlabs/PWC-Net/tree/master/PyTorch

**Comment by lliuz** [comment]
Hi @luvegood, thanks for your reply.
I have struggled to train PyTorch model for a few days but has almost no progress. EPE will hardly drop during training. I have tried to adapt the training code from `ClementPinard/FlowNetPytorch` with modifications of training details to match your paper, and I have carefully compared with your Caffe version but does not work. 
Has anyone successfully trained the version of PyTorch or have any suggestions?  I
